mmengine/hooks/param_scheduler_hook.py

```diff
@@ -32,15 +35,15 @@ def after_train_iter(self,
                 keep ``data_batch`` here.
         """
 
-        def step(param_schedulers):
-            assert isinstance(param_schedulers, list)
-            for scheduler in param_schedulers:
-                if not scheduler.by_epoch:
-                    scheduler.step()
-
         if runner.param_schedulers is None:
             return
 
+        def step(_param_schedulers):
+            assert isinstance(_param_schedulers, list)
+            for scheduler in _param_schedulers:
+                if not scheduler.by_epoch:
+                    scheduler.step()
```

Comment:
Why do we need to change the position of `step` method?

Reply:
Got it. It is unnecessary to rename `param_schedulers` to `_param_schedulers` in step method.

Reply:
I will change `_param_schedulers` back to `param_schedulers`.